airflow/decorators/__init__.py

```diff
@@ -15,129 +15,58 @@
 # specific language governing permissions and limitations
 # under the License.
 
-from typing import Callable, Dict, Iterable, List, Optional, Union
-
-from airflow.decorators.python import python_task
-from airflow.decorators.python_virtualenv import _virtualenv_task
+from airflow.decorators.python import PythonDecoratorMixin
```

Comment:
```suggestion
from airflow.decorators.python import PythonDecoratorMixin, python_task
```

We probably have to keep this for back-compat reasons.